content/docs/recipes/upgrade-kafka.md

```diff
@@ -0,0 +1,289 @@
+    + [Preparing for the upgrade](#preparing-for-the-upgrade)
+      - [5. Verify the inter.broker.protocol.version is matching the current OperatorVersion](#5-verify-the-interbrokerprotocolversion-is-matching-the-current-operatorversion)
+      - [6. Update the inter.broker.protocol.version to match the current version](#6-update-the-interbrokerprotocolversion-to-match-the-current-version)
+      - [7. Install the new OperatorVersion using --skip-instance](#7-install-the-new-operatorversion-using---skip-instance)
+      - [8. Verify the new OperatorVersion is installed correctly](#8-verify-the-new-operatorversion-is-installed-correctly)
+    + [Upgrade the KUDO Kafka Instance](#upgrade-the-kudo-kafka)
+      - [9. Upgrade the KUDO Kafka Instance](#9-upgrade-the-kudo-kafka)
+      - [10. Verify the Kafka version through the pod logs](#10-verify-the-kafka-version-through-the-pod-logs)
+      - [11. Verify the Kafka version through the container version](#11-verify-the-kafka-version-through-the-container-version)
+      - [12. Verify the Kafka version through the app version of the installed KUDO Instance](#12-verify-the-kafka-version-through-the-app-version-of-the-installed-kudo-instance)
+      - [13. (Optional) Bump the `inter.broker.protocol.version` to upgraded version](#13--optional--bump-the--interbrokerprotocolversion--to-upgraded-version)
+  * [Future improvements](#future-improvements)
+
+## Requirements
+
+- Kubernetes cluster with KUDO version > 0.10.0 installed
+- Have a KUDO Kafka cluster version 1.1.0 up and running in the namespace `kudo-kafka`
+- Have binaries of `jq` and `grep` installed in the `$PATH`
+
+## Steps
+
+### Verifying if KUDO Kafka is ready for the upgrade
+
+#### 1. Get the KUDO Kafka Instance object name
+
+Verify the KUDO Kafka instance object is present in the expected namespace
+
+```bash
+kubectl get instances -n kudo-kafka
+```
+
+expected output are the KUDO Instance objects present in the namespace `kudo-kafka`:
+
+```bash
+NAME    AGE
+kafka   71m
+zk      130m
+```
+
+#### 2. Verify the KUDO Kafka instance plans are COMPLETE 
+
+```bash
+kubectl kudo plan status --instance=kafka -n kudo-kafka
+```
+
+expected output is the plan status for instance `kafka`:
+
+```
+Plan(s) for "kafka" in namespace "kudo-kafka":
+.
+└── kafka (Operator-Version: "kafka-1.1.0" Active-Plan: "deploy")
+    ├── Plan deploy (serial strategy) [COMPLETE]
+    │   └── Phase deploy-kafka (serial strategy) [COMPLETE]
+    │       └── Step deploy [COMPLETE]
+    ├── Plan mirrormaker (serial strategy) [NOT ACTIVE]
+    │   └── Phase deploy-mirror-maker (serial strategy) [NOT ACTIVE]
+    │       └── Step deploy [NOT ACTIVE]
+    └── Plan not-allowed (serial strategy) [NOT ACTIVE]
+        └── Phase not-allowed (serial strategy) [NOT ACTIVE]
+            └── Step not-allowed [NOT ACTIVE]
+```
+
+#### 3. Get the operatorversion  of KUDO Kafka Instance
```

Comment:
Use English for the concept of Operator Version, then `operator version` for the commands.